test/functional/api/v1/dependencies_controller_test.rb

```diff
@@ -31,6 +31,23 @@ class Api::V1::DependenciesControllerTest < ActionController::TestCase
     end
   end
 
+  # INVALID GEMS:
+  context "On GET to index --> with invalid gems --> JSON" do
```

Comment:
can we please be more specific, perhaps "with hash in gems params"?

Reply:
sure

Reply:
Would be "malformed" enough?